part1/simple_config.rst

```diff
@@ -251,7 +251,7 @@ So, we can simply run gem5 from the root gem5 directory as:
 
```

Comment:
How about "So, we can simply run gem5 from the root gem5 directory as shown below. ``configs/tutorial/simple.py`` is the path to the config file we wrote above.